mypy_django_plugin/transformers/models.py

```diff
@@ -362,91 +369,97 @@ def run_with_model_cls(self, model_cls: Type[Model]) -> None:
+                continue
 
             if isinstance(relation, OneToOneRel):
                 self.add_new_node_to_model_class(attname, Instance(related_model_info, []))
                 continue
 
-            if isinstance(relation, (ManyToOneRel, ManyToManyRel)):
-                related_manager_info = None
-                try:
-                    related_manager_info = self.lookup_typeinfo_or_incomplete_defn_error(
-                        fullnames.RELATED_MANAGER_CLASS
-                    )  # noqa: E501
-                    default_manager = related_model_info.names.get("_default_manager")
-                    if not default_manager:
-                        raise helpers.IncompleteDefnException()
-                except helpers.IncompleteDefnException as exc:
-                    if not self.api.final_iteration:
-                        raise exc
-                    else:
-                        if related_manager_info:
-                            """
-                            If a django model has a Manager class that cannot be
-                            resolved statically (if it is generated in a way
-                            where we cannot import it, like `objects = my_manager_factory()`),
-                            we fallback to the default related manager, so you
-                            at least get a base level of working type checking.
-
-                            See https://github.com/typeddjango/django-stubs/pull/993
-                            for more information on when this error can occur.
-                            """
-                            self.add_new_node_to_model_class(
-                                attname, Instance(related_manager_info, [Instance(related_model_info, [])])
-                            )
-                            related_model_fullname = related_model_cls.__module__ + "." + related_model_cls.__name__
-                            self.ctx.api.fail(
-                                f"Couldn't resolve related manager for relation {relation.name!r} (from {related_model_fullname}.{relation.field}).",
-                                self.ctx.cls,
-                                code=MANAGER_MISSING,
-                            )
-                        continue
-
-                # Check if the related model has a related manager subclassed from the default manager
-                # TODO: Support other reverse managers than `_default_manager`
-                default_reverse_manager_info = self.get_reverse_manager_info(
-                    model_info=related_model_info, derived_from="_default_manager"
-                )
-                if default_reverse_manager_info:
-                    self.add_new_node_to_model_class(
-                        attname, Instance(default_reverse_manager_info, []), no_serialize=True
-                    )
-                    continue
+            if not isinstance(relation, (ManyToOneRel, ManyToManyRel)):
```

Comment:
```suggestion
            elif not isinstance(relation, (ManyToOneRel, ManyToManyRel)):
```

Reply:
I applied all `if -> elif` changes, could you elaborate on why is it important? Is it performance or readability issue?

Reply:
Readability, perhaps personally. But I realised I'm looking at it trying to find cases entering multiple if-branches. But then realise they all do continue